src/nidcpower/metadata/config_addon.py

```diff
@@ -1,5 +1,6 @@
 # We need to maintain the version here since it needs to be updated by the build process on GitHub
 config_additional_config = {
+    'custom_types': [],
```

Comment:
please add TODO pointing to Issue for adding these

Reply:
I've filed an issue for this and added a TODO comment referencing that issue in the latest commit.